includes/assets/stories.css

```diff
@@ -372,6 +375,15 @@
   transform: rotateZ(-45deg);
 }
 
+/*
+ * amp-story-player's 'shadow-root-intermediary' element shows black space on top,
+ * because of the 'a' links. Making 'a' absolutely positioned removes them from normal
+ * relative flow, removes the space.
+ */
```

Comment:
Interesting! Maybe something to report upstream?

Reply:
though this issue comes on non-AMP pages only due to the absence of `css/ampshared.css`.